client/web/compose/src/components/PageBlocks/Navigation/Configurator.vue

```diff
@@ -0,0 +1,291 @@
+                buttons
+                button-variant="outline-secondary"
+                size="sm"
+                name="alignment"
+                :options="alignmentOptions"
+              />
+            </b-form-group>
+          </b-col>
+
+          <b-col
+            sm="4"
+            class="mb-2 mb-sm-0"
+          >
+            <b-form-group
+              horizontal
+              :label="$t('navigation.fillJustify')"
+            >
+              <b-form-radio-group
+                v-model="options.display.fillJustify"
+                buttons
+                button-variant="outline-secondary"
+                size="sm"
+                :options="fillJustifyOptions"
+              />
+            </b-form-group>
+          </b-col>
+        </b-row>
+      </div>
+
+      <div class="mb-3 mt-2">
+        <div class="d-flex align-items-center mb-4">
+          <h5 class="text-primary mb-0">
+            {{ $t("navigation.listItems") }}
+          </h5>
+
+          <b-button
+            variant="link"
+            class="d-flex align-items-center text-decoration-none"
+            @click="addNavigationItem"
+          >
+            <font-awesome-icon
+              :icon="['fas', 'plus']"
+              size="sm"
+              class="mr-1"
+            />
+            {{ $t("navigation.add") }}
+          </b-button>
+        </div>
+
+        <div class="mt-3">
+          <draggable
+            v-model="block.options.lists"
+            group="sort"
+            handle=".grab"
+          >
+            <div
+              v-for="(column, index) in block.options.lists"
+              :key="index"
+            >
+              <hr v-if="index">
+
+              <b-table-simple
+                borderless
+                class="nav-display-options-lists"
+              >
+                <thead>
+                  <tr>
+                    <th style="width: auto" />
+                    <th>
+                      {{ $t("navigation.type") }}
+                    </th>
+                    <th>
+                      {{ $t("navigation.color") }}
+                    </th>
+                    <th>
+                      {{ $t("navigation.background") }}
+                    </th>
+                    <th class="text-center">
+                      {{ $t("navigation.enabled") }}
+                    </th>
+                    <th style="width: auto;min-width: 100px" />
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td class="align-middle">
+                      <font-awesome-icon
+                        :icon="['fas', 'bars']"
+                        class="grab text-grey"
+                      />
+                    </td>
+                    <td class="align-middle">
+                      <b-form-select
+                        v-model="column.type"
+                        :options="navigationItemTypes"
+                        size="sm"
```

Comment:
To make it look consistent (vue-select problems), just change all inputs and buttons sizes to 'md'. Remember bootstrap things default to md, others might not. Same for the NavTypes

Reply:
I see you changed them, but like I said above, bootstrap inputs default to md!
Also the radio group should stay sm